src/bluesearch/database/article.py

```diff
@@ -630,14 +686,23 @@ class TEIXMLParser(ArticleParser):
 
     Parameters
     ----------
-    xml_content
-        The content of a TEI XML file.
+    path
+        The path to a TEI XML file.
+    is_arxiv
+        Set to `True` if the TEI XML file was generated by parsing an arXiv PDF.
     """
 
-    def __init__(self, xml_content: str):
-        self.content = ElementTree.fromstring(xml_content)
+    def __init__(self, path: str | Path, is_arxiv: Optional[bool] = False):
+        path = Path(path)
+        with path.open() as fp:
+            self.content = ElementTree.fromstring(fp.read())
         self.tei_namespace = {"tei": "http://www.tei-c.org/ns/1.0"}
         self._tei_ids: dict[str, str] | None = None
+        if is_arxiv:
+            arxiv_id = get_arxiv_id(path)
+            self._arxiv_id = arxiv_id
```

Comment:
Why don't we write `self._arxiv_id = get_arxiv_id(path)`?

Reply:
In fact the whole block 
https://github.com/BlueBrain/Search/blob/4089e1756f9de6acbe26d2a7707a2a292659e80a/src/bluesearch/database/article.py#L701-L705
can be replaced by a one-line conditional expression.

Changed in 078ce32.